mozregression/main.py

```diff
@@ -247,10 +240,12 @@ def cli(argv=None):
         # can go to inbound from nightly.
         fetch_config.set_inbound_branch(options.inbound_branch)
 
-    if options.inbound:
+    # bisect inbound if last good revision and first bad revision are set
+    if (options.first_bad_revision and options.last_good_revision):
```

Comment:
You have to use a "or" condition here instead of "and". If there is a missing rev option (--good-rev or --bad-rev) it wil be handled in the bisect_inbound function (https://github.com/mozilla/mozregression/blob/9a99d82071f383147b14899b64c96bad467e6bd6/mozregression/main.py#L217).
if you don't do that, a user that only use "mozregression --good-rev 123" for example will default to nightly bisection instead of inbound which is not really user-friendly.